int16 gear
float64 speed_mps
float64 steer_rad
int16 brake